src/proxy-ui-api/frontend/tests/e2e/pages/tabs/settingsTab.js

```diff
@@ -41,7 +42,44 @@ const settingsTabCommands = {
   },
 };
 
-const systemParametersCommands = {};
+const systemParametersTabCommands = {
+  openTimeStampingAddDialog: function () {
+    this.click('@timestampingAddButton');
+    this.waitForElementVisible('@timestampingAddDialogServiceSelection');
+    return this;
+  },
+  openTimeStampingDeleteDialog: function () {
+    this.waitForElementVisible('@timestampingDeleteButton');
+    this.click('@timestampingDeleteButton');
+    this.waitForElementVisible('@timestampingDeleteDialog');
+    return this;
+  },
+  deleteCurrentTimestampingService: function () {
+    this.click('@timestampingDeleteButton');
+    this.waitForElementVisible('@timestampingDeleteDialogSaveButton');
+    this.click('@timestampingDeleteDialogSaveButton');
+
+    this.waitForElementNotPresent('@timestampingDeleteDialog');
+    this.waitForElementNotPresent('@timestampingServiceTableRow');
+    return this;
+  },
+
+  assertTimestampingTableContents: function (expectedName, expectedUrl) {
+    // Service table row is visible
+    this.waitForElementVisible('@timestampingServiceTableRow');
+    // Delete button is visible in row
+    this.waitForElementVisible('@timestampingDeleteButton');
+    this.getText('@timestampingTableFirstCell', function (foundName) {
+      assert.equal(foundName.value, expectedName);
+    });
+    this.getText('@timestampingTableSecondCell', function (foundUrl) {
+      console.log(foundUrl.value);
```

Comment:
Is this for debugging purposes? Probably not needed

Reply:
True, removed now.